Basic Select/Weather-Observation-Station-5.sql

```diff
@@ -1,3 +1,3 @@
 # Author: Thomas George Thomas
 select city,length(city) from station order By length(city) asc, city asc limit 1;
-select distinct(City),length(city) from station order by length(city) desc, city asc limit 1;
\ No newline at end of file
+select distinct(City),length(city) from station order by length(city) desc, city asc limit 1;
```

Comment:
Could you please delete this file and pull it again since there's no change in the file. This way it will not show a change in the PR. Thanks!